eisbuk-admin/src/translations/en.json

```diff
@@ -68,7 +68,9 @@
     "DeleteSlot": "Are you sure you want to delete the",
     "NonReversible": "This action is non-reversible",
     "SendEmailTitle": "Confirm email",
-    "ConfirmEmail": "Are you sure you want to send an email to:"
+    "ConfirmEmail": "Are you sure you want to send an email to:",
```

Comment:
We should not assume a specific structure of the message (like here `fixedText` `customerName``?`).
[We should use interpolation](https://www.i18next.com/translation-function/interpolation) to leave a spot in the translation string that will be filled in when invoking `translate`.